src/index.js

```diff
@@ -0,0 +1,74 @@
+module.exports = function (_context, options) {
+  const {
+    accountId,
+    baseUrl = "https://app.papercups.io",
+    greeting,
+    newMessagePlaceholder,
+    primaryColor = "#1890ff",
+    requireEmailUpfront,
+    showAgentAvailability,
+    subtitle,
+    title,
+  } = options;
```

Comment:
should we make this the same as https://github.com/reichert621/snippet/blob/master/src/index.tsx#L10-L40 ?

Reply:
Added the rest of the properties @reichert621! I excluded the ones that have functions as values because they can't be injected into the html. I also excluded `styles` since that property is React specific.

Reply:
perfect!